students/arkadiusz_kasprzyk/lesson_08_regular_expressions/valid_email.py

```diff
@@ -0,0 +1,26 @@
+# valid_email.py
+"""
+https://isemail.info/about
+https://tools.ietf.org/html/rfc5321
+so it's virtually impossible to do it properly...
+only simplest rules checked
+"""
+
+import re
+
+
+def valid_email(email):
+    """
+    Examples
+    --------
+    valid_email('ab.cd@c.d')
+    """
+
+    if not re.match(r'.{5,254}', email):
+        raise ValueError('e-mail address cannot exceed 254 characters and cannot be shorter then 5.')
+
+    regex = re.compile ( r'^(\w+[.-])*(\w+)@(\w+[.-])*(\w+)\.(\w+)$', re.IGNORECASE )
```

Comment:
It should actually be a bit more complicated, for example `+` and `_` are valid characters. For production code look for "regex library" on Google, for an exercise this is good enough.